src/canister_tests/src/api.rs

```diff
@@ -51,3 +51,86 @@ pub fn lookup(
         Err(e) => panic!("Failed to lookup: {:?}", e),
     }
 }
+
+pub fn get_anchor_info(
+    env: &StateMachine,
+    canister_id: CanisterId,
+    sender: PrincipalId,
+    user_number: types::UserNumber,
+) -> Result<types::IdentityAnchorInfo, CallError> {
+    match framework::call_candid_as(env, canister_id, sender, "get_anchor_info", (user_number,)) {
+        Ok((r,)) => Ok(r),
+        Err(e) => Err(e),
+    }
+}
+
+pub fn enter_device_registration_mode(
+    env: &StateMachine,
+    canister_id: CanisterId,
+    sender: PrincipalId,
+    user_number: types::UserNumber,
+) -> Result<types::Timestamp, CallError> {
+    match framework::call_candid_as(
+        env,
+        canister_id,
+        sender,
+        "enter_device_registration_mode",
+        (user_number,),
+    ) {
+        Ok((r,)) => Ok(r),
+        Err(e) => Err(e),
+    }
```

Comment:
```suggestion
    framework::call_candid_as(
        env,
        canister_id,
        sender,
        "enter_device_registration_mode",
        (user_number,)).map(|(x,)| x)
```

and same above and below